src/gwc/pom.xml

```diff
@@ -85,6 +85,10 @@
       <groupId>org.geowebcache</groupId>
       <artifactId>gwc-diskquota-jdbc</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.geowebcache</groupId>
+      <artifactId>gwc-aws-s3</artifactId>
```

Comment:
quite a bit:
```
[     ] +- org.geowebcache:gwc-aws-s3:jar:1.8-SNAPSHOT:compile
[544K ] |  \- com.amazonaws:aws-java-sdk-s3:jar:1.9.30:compile
[247K ] |     \- com.amazonaws:aws-java-sdk-kms:jar:1.9.30:compile
[488K ] |        \- com.amazonaws:aws-java-sdk-core:jar:1.9.30:compile
[577K ] |           +- org.apache.httpcomponents:httpclient:jar:4.3.4:compile
[276K ] |           |  \- org.apache.httpcomponents:httpcore:jar:4.3.2:compile
[894K ] |           +- com.fasterxml.jackson.core:jackson-databind:jar:2.3.2:compile
[ 35K ] |           |  +- com.fasterxml.jackson.core:jackson-annotations:jar:2.3.0:compile
[195K ] |           |  \- com.fasterxml.jackson.core:jackson-core:jar:2.3.2:compile
[608K ] |           \- joda-time:joda-time:jar:2.8.1:compile (version selected from constraint [2.2,))

[3864K total]
```
Yet making it a plugin would add some unneeded complexity esp. to the UI.
I'll ask on the mailing list.